packages/react-native-reanimated/src/layoutReanimation/web/transition/EntryExit.web.ts

```diff
@@ -0,0 +1,52 @@
+'use strict';
+import type { AnimationData, TransitionData } from '../animationParser';
+import { AnimationsData } from '../config';
+
+export function EntryExitTransition(
+  name: string,
+  transitionData: TransitionData
+) {
+  const exitingAnimationData = structuredClone(
+    AnimationsData[transitionData.exiting]
+  );
+  const enteringAnimationData = structuredClone(
+    AnimationsData[transitionData.entering]
+  );
+
+  const additionalData = {
+    translateX: `${transitionData.translateX}px`,
+    translateY: `${transitionData.translateY}px`,
+    scale: `${transitionData.scaleX},${transitionData.scaleY}`,
+  };
+
+  const keyframeData: AnimationData = {
+    name,
+    style:
+      '0' in exitingAnimationData.style
+        ? {}
+        : { 0: { transform: [additionalData] } },
+    duration: 300,
+  };
```

Comment:
Let's think about whether we still need this

Reply:
Removed in [25fbb28](https://github.com/software-mansion/react-native-reanimated/pull/6232/commits/25fbb283ac85afae124c08a801f92e8c9482823d) since it is no longer required